lucet-runtime/lucet-runtime-tests/src/guest_fault.rs

```diff
@@ -305,8 +305,8 @@ macro_rules! guest_fault_tests {
                 _siginfo_ptr: *const siginfo_t,
                 _ucontext_ptr: *const c_void,
             ) -> SignalBehavior {
-                // Triggered by a SIGSEGV writing to protected page
-                assert!(signum == SIGSEGV);
+                // Triggered by a SIGSEGV writing to protected page, or SIGBUS on macos
+                assert!(signum == SIGSEGV || signum == SIGBUS);
```

Comment:
Could we make this more precise with something like:
```rust
#[cfg(not(target_os="macos"))]
const PROTECTED_PAGE_SIG: libc::c_int = SIGSEGV;
#[cfg(target_os="macos")]
const PROTECTED_PAGE_SIG: libc::c_int = SIGBUS;
[...]
assert!(signum == PROTECTED_PAGE_SIG);
```

Reply:
Done - there are instances we need the `libc::c_int`, and instances we need the `nix::Signal`, so this turned into two `const`s